packages/vscode-extension-serverless-workflow-editor/it-tests/serverless-workflow-editor-extension-syntax-highlight.test.ts

```diff
@@ -0,0 +1,92 @@
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as path from "path";
+import { expect } from "chai";
+import { By, WebDriver, WebElement } from "vscode-extension-tester";
+import VSCodeTestHelper from "./helpers/VSCodeTestHelper";
+import SwfTextEditorTestHelper from "./helpers/swf/SwfTextEditorTestHelper";
+
+describe("Serverless workflow editor - syntax highlighting test", () => {
+  const TEST_PROJECT_FOLDER: string = path.resolve("it-tests-tmp", "resources", "syntax-highlight");
+
+  let testHelper: VSCodeTestHelper;
+  let driver: WebDriver;
+
+  before(async function () {
+    this.timeout(30000);
+    testHelper = new VSCodeTestHelper();
+    await testHelper.openFolder(TEST_PROJECT_FOLDER, "syntax-highlight");
+  });
+
+  beforeEach(async function () {
+    await testHelper.closeAllEditors();
+    await testHelper.closeAllNotifications();
+  });
+
+  afterEach(async function () {
+    this.timeout(15000);
+    await testHelper.closeAllEditors();
+    await testHelper.closeAllNotifications();
+  });
+
+  it("Checks syntax highlighting of *.sw.json files in serverless workflow editor", async function () {
+    this.timeout(30000);
+
+    const editorWebviews = await testHelper.openFileFromSidebar("syntax-highlight-hello-world.sw.json");
+    const swfTextEditor = new SwfTextEditorTestHelper(editorWebviews[0]);
+    const textEditor = await swfTextEditor.getSwfTextEditor();
+
+    driver = textEditor.getDriver();
+
+    // get color codes of json properties
+    const property1WebElement = await getWebElementByDisplayedText("specVersion");
+    const property2WebElement = await getWebElementByDisplayedText("states");
+    const property1Color = await resolveWebElementColor(property1WebElement);
+    const property2Color = await resolveWebElementColor(property2WebElement);
+
+    // get color codes of json string values
+    const stringValue1WebElement = await getWebElementByDisplayedText("helloworld");
+    const stringValue2WebElement = await getWebElementByDisplayedText("HelloWorldWorkflow");
+    const stringValue1Color = await resolveWebElementColor(stringValue1WebElement);
+    const stringValue2Color = await resolveWebElementColor(stringValue2WebElement);
+
+    // get color code of json boolean value
+    const booleanValueWebElement = await getWebElementByDisplayedText("true");
+    const booleanValueColor = await resolveWebElementColor(booleanValueWebElement);
+
+    // check that properties have the same color
+    expect(property1Color == property2Color, "All properties should be the same color").to.be.true;
```

Comment:
Could you please use 
```suggestion
    expect(property1Color,  "All properties should be the same color").equal(property2Color).to.be.true;
```